### PR TITLE
Sidebar view translation

### PR DESCRIPTION
**Pull request by jiteshjha**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] All tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request are descriptively named
* [x] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer
* [x] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by jiteshjha** [comment]
Hello @jywarren!
Sidebar view I18n is ready to be merged. 
Thanks!

File: app/views/sidebar/_notes.html.erb [hunk 1]
```diff
@@ -13,15 +13,14 @@
...
-            by <a href="/profile/<%= note.author.name %>"><%= note.author.name %></a>
+            <%= raw t('sidebar._notes.updated_by', :time => time_ago_in_words(note.created_at), :url1 => "/profile/"+note.author.name, :author_name => note.author.name) %>
           </small>
         </p>
       <% end %>
     <% end %>
 
     <% if params[:controller] == "search" %>
-      <p><a href="/tag/<%= params[:id] %>">More...</a></p>
+      <p><a href="/tag/<%= params[:id] %>"><%= t('sidebar._notes.more') %>...</a></p>
```
Comment (jywarren) [inline app/views/sidebar/_notes.html.erb]: Lets put the `...` inside, as not all languages use that notation.

**Comment by jiteshjha** [comment]
I have made the requested changes -- any suggestions?

**Comment by jiteshjha** [comment]
Oops! I accidently closed this PR.
I have made the requested changes -- any suggestions?

**Comment by jywarren** [comment]
Done! Thanks!

**Pull request merged**